src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -352,6 +352,14 @@ void GetZoneEquipmentData(EnergyPlusData &state)
     } // end loop over controlled zones
     for (int controlledSpaceLoop = 1; controlledSpaceLoop <= numControlledSpaces; ++controlledSpaceLoop) {
         CurrentModuleObject = "SpaceHVAC:EquipmentConnections";
+        if (!state.dataHeatBal->doSpaceHeatBalanceSimulation) {
+            ShowWarningError(
+                state,
+                format("{} requires \"Do Space Heat Balance for Simulation = Yes\" in ZoneAirHeatBalanceAlgorithm. {} objects will be ignored.",
+                       CurrentModuleObject,
+                       CurrentModuleObject));
+            break;
+        }
```

Comment:
So if the ZoneAirHeatBalanceAlgorithm object is missing, or is present and Do Space Heat Balance for Simulation = No, and the inputs include any of the SpaceHVAC objects, then those space objects will be ignored and the simulation continues. This does sound reasonable since the space heat balance will not be executed and therefore no data available to assign to the space reporting. I was thinking if the SpaceHVAC inputs were NOT autosized then the simulation should still account for spaces, but that can only happen when Do Space Heat Balance for Simulation = Yes. Changes are Approved!